rds_test.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+)
+
+var AWSDBExampleAlert = `
+{
+	"type": "aws_db_instance",
+	"depends_on": [
+		"aws_db_subnet_group.postgres",
+		"aws_security_group.postgres",
+		"local.terraform"
+	],
+	"primary": {
+		"id": "au-identity-equifax",
+		"attributes": {
+			"tags.Name": "au-identity-equifax",
+			"tags.alert": "below 10 pulse in 30: above 2000 pulse in 30",
```

Comment:
There's a colon character separating the alerts. It should either be a semi-colon or a comma. I think maybe semi-colon is good

Reply:
`Tag values may only contain unicode letters, digits, whitespace, or one of these symbols: _ . : / = + - @`

I forgot to let you know but Tag doesn't accept comma or semi-colon.